homeassistant/components/esphome/strings.json

```
@@ -46,6 +46,15 @@
     },
     "flow_title": "{name}"
   },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "allow_service_calls": "Allow the device to make arbitrary Home Assistant service calls."
```

I don't think the word "arbitrary" is necessary? If we do want to keep it in, then let's use the word "custom".

```suggestion
          "allow_service_calls": "Allow the device to make Home Assistant service calls."
```